scripts/helpers/buildAuthorBio.js

```diff
@@ -0,0 +1,35 @@
+export const buildAuthorBio = ({ name, title, image, description }) => {
+  const authorBio = document.createElement("div");
+  authorBio.classList.add("author-bio");
+  const authorInfo = document.createElement("div");
```

Comment:
can we move the `author-meta__info` declaration lower in the file, below it's block parent, `author-meta`, declaration?